lib/ansible/modules/cloud/amazon/ecs_service_info.py

```diff
@@ -237,7 +245,10 @@ def main():
     else:
         ecs_facts = task_mgr.list_services(module.params['cluster'])
 
-    module.exit_json(changed=False, ansible_facts=ecs_facts, **ecs_facts)
+    if is_old_facts:
+        module.exit_json(changed=False, ansible_facts=ecs_facts, **ecs_facts)
```

Comment:
Minor nit, but is it worth changing the variable name too?

Reply:
In the long run, definitely. But if I have to change all internal stuff in all the ~110 module renames coming up (given the time constraints), this will dramatically increase the chance of accidentally breaking things, so I wanted to avoid doing that.

Reply:
Changed in 774b0c2b3033f35b46bca16703379e23273346b5